.cirrus.yml

```diff
@@ -736,6 +736,33 @@ upgrade_test_task:
     always: *logs_artifacts
 
 
+image_build_task:
+    name: "Build multi-arch images and manifest lists"
+    alias: image_build
+    skip: *tags
+    # FIXME: Run nightly on 'main' branch
+    # only_if: $CIRRUS_CRON == 'multiarch'
+    only_if: $CIRRUS_CHANGE_TITLE =~ '.*CI:BUILD.*'
+    depends_on:
+        - ext_svc_check
+    timeout_in: 120m  # emulation is sssllllooooowwww
+    gce_instance:
+        image_project: libpod-218412
+        zone: "us-central1-a"
+        # More muscle required for parallel multi-arch build
+        type: "n2-standard-4"
+        disk: 200
+        # FIXME: Use "build-push-c${IMAGE_SUFFIX}"
+        image_name: build-push-c4664087534632960
+    env:
+        # FIXME: Add encrypted push credentials
+        DRYRUN: 1
+    # FIXME: Build all 3 podmanimage + helloimage
+    script:
+        - set -a; source /etc/automation_environment; set +a
```

Comment:
`set -a` is an uncommon flag; could you document what it means and why it's needed? (i.e., if there's some important reason why that file doesn't use `export` on variables)?

Reply:
Ah, an easy question!  It's that way because I never fixed it :grin:  The fix would [go right here](https://github.com/containers/automation/blob/f5c59a92ef62d9e1c0934021994368c717b54119/bin/install_automation.sh#L128-L132) but the hard part is then building new VM images and testing them in every damn repo. CI system :disappointed: 

so...yeah...easier to just sprinkle some `set -a` around and bury head in sand.